### ISSUE TITLE
[3.x]: Uncaught TypeError when adding a Matrix Block in Live Preview

### ISSUE THREAD

**Issue by tommysvr** [craftcms/cms#12992]
### Description
When adding a block on a Matrix field in Live Preview, if you have block types to display the 'Add a Block' dropdown, an `Uncaught TypeError` occurs.

### Steps to reproduce

1. Add a Matrix field with enough blocks to display the 'Add a Block' dropdown button
2. Open Live Preview
3. Click 'Add a Block' -> select a block type
4. No block will be added, dropdown button becomes unclickable and there will be an error in the browser console

### Expected behavior
A block is added without issue.

### Actual behavior
The error is:
```
Uncaught TypeError: Cannot read properties of null (reading 'get')
    at Object.<anonymous> (UiLayerManager.js:79:27)
    at Function.each (jquery.js?v=1679980140:2:3002)
    at w.fn.init.each (jquery.js?v=1679980140:2:1482)
    at s.constructor.getLayerIndex (UiLayerManager.js:78:20)
    at s.constructor.removeLayer (UiLayerManager.js:63:31)
    at s.constructor.hide (CustomSelect.js:230:7)
    at s.constructor.hideMenu (MenuBtn.js:241:17)
    at s.constructor.onMouseDown (MenuBtn.js:214:14)
    at s.constructor.<anonymous> (Base.js:153:23)
    at HTMLButtonElement.o (jquery.js?v=1679980140:2:89240)
```

I'm not able to reproduce on Craft 4.4.5

### Craft CMS version

3.8.5

### Installed plugins and versions

-

**Comment by i-just** [comment]
Hi Tom, thanks for reporting. I submitted a PR to address this.

**Comment by brandonkelly** [comment]
Fixed for the next release via #12997.

**Comment by brandonkelly** [comment]
Craft 3.8.6 and 4.4.6 are out with that fix 🎉
